portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,110 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+h1{
+    
+  margin-left: auto;
+  margin-right: auto;
+  margin-top:200px;
+  
+  color:rgba(43,142,190);
+  text-align: center;
+  text-shadow: 0 0 18px rgb(255,255,255);
+  font-family:  'Proxima Nova Soft', 'Helvetica Neue', sans-serif;;
+}
+
+p {
+    
+  margin-left: auto;
+  margin-right: auto;
+  margin-top:10px;
+  
+  color:rgba(43,142,190);
+  text-align: center;
+  background-color: rgba(255, 255, 255, 0.7); 
+  width:400px;
+  padding:10px;
+  font-family: 'Courier New', Courier, monospace
+}
+
+body {
+  background-color: floralwhite;
+  background-image: url("images/bg2.JPG");
+  background-size: cover
+}
+
+div{
+  margin-left: auto;
```

Comment:
I like how youve organized this block. I'd take another pass through this CSS and see if you can make it cleaner by:
1) grouping related properties (font.*, margin.*, top|left|bottom|right, etc)
2) Having a consistent ordering of properties between blocks (if you have position categories at the top in one, do it for all)
3) Using the same color format across the board (rgba is a great choice)
4) always have space between the colon and the property value
5) add semicolons to the end of every line, including the last one (this will make it easier to add properties in the future.